lib/cisco_node_utils/cmd_ref/interface.yaml

```diff
@@ -16,6 +16,8 @@ access_vlan:
 
 all_interfaces:
   multiple:
+  ios_xr:
+    get_command: 'show running all-interfaces'
```

Comment:
In XR, fully unconfigured interfaces do not show up in 'show run' but they do show up in 'show run all-interfaces'. This was causing our interface discovery logic in ciscotest.rb to fail to find any interfaces on my node that it could use - this change fixed that.